calico-enterprise/_includes/components/ReqsSys.js

```diff
@@ -48,6 +49,28 @@ function NodeRequirementsEnt(props) {
             )}
           </ul>
         </li>
+        <li>
+          <p>
+            If your node is running RHEL 7 or RHEL 8, you must install a specialized policy package before you install {prodname}.
+            With this package, {prodname} can use SELinux contexts in a series of rules that allow it to interact with persistent and ephemeral data in nonstandard host system locations.
+          </p>
+          <ul>
+            <li>
+              <p>If your node has RHEL 7 installed, then run the following command:</p>
+              <CodeBlock language="bash">
+                {`. /etc/os-release
```

Comment:
I would probably choose to use `source` rather than `.` to make it more visible and clear what's going on here.

Reply:
There is no need for the `. /etc/os-release` step anymore as it is explicit in your dnf command. Remove this and use the `dnf` command.

Reply:
Another option is to not show our customer the detailed `dnf` step but provide them with RPM download links. It can either by in a sentence or in a table. Most system administrators know how to install a RPM I think. WDYT?

Reply:
I'm removing the ./etc/os-release line and leaving the plain dnf line.